client/me/purchases/manage-purchase/index.jsx

```diff
@@ -906,6 +911,8 @@ class ManagePurchase extends Component {
 						getManagePurchaseUrlFor={ getManagePurchaseUrlFor }
 						isProductOwner={ isProductOwner }
 						getAddNewPaymentMethodUrlFor={ getAddNewPaymentMethodUrlFor }
+						purchaseIsInAppPurchase={ purchaseIsInAppPurchase }
+						purchaseManagementIsLocked={ purchaseManagementIsLocked }
```

Comment:
You already pass a `purchase` prop to the `PurchaseNotice` component, so the component can figure out the two flags by looking at `purchase`. Doesn't need two new props.

Reply:
Good idea, I'll do that!